### ISSUE TITLE
Accessibility: WCAG 1.1.1 – NON-TEXT CONTENT

### ISSUE THREAD

**Issue by andreamartz** [andreamartz/mcc-authentic-grace-2.0#21]
https://www.boia.org/WCAG2/CP/1.1.1

**Comment by andreamartz** [comment]
1. Modal link "Sign me up!" should be a button using the <button> tag.
2. It may also be a problem that there is no close button on the modal.
3. Pair the focus state with the hover state on the modal buttons, including the close button.  (see "INCLUDING FOCUS STATES" in https://www.smashingmagazine.com/2014/09/making-modal-windows-better-for-everyone/
